symfony/plugins/orangehrmTimePlugin/Controller/EditTimesheetController.php

```diff
@@ -67,10 +69,14 @@ public function isCapable(Request $request): bool
                 ->getTimesheetDao()
                 ->getTimesheetById($request->attributes->getInt('id'));
             if ($timesheet instanceof Timesheet) {
+                if ($this->getUserRoleManagerHelper()->isSelfByEmpNumber($timesheet->getEmployee()->getEmpNumber())
+                    && $timesheet->getState() === 'APPROVED') {
```

Comment:
Shall we remove `$timesheet->getState() === 'APPROVED'` for the moment. Will address it under `OHRM5X-1056`

Reply:
done